git/intermediate_git/working_with_remotes.md

```diff
@@ -0,0 +1,89 @@
+
+<span id='force-with-lease'>It is worth giving special mention to `git push --force-with-lease`</span>, a command which in some companies is the default option. The reason for this is that it's a fail-safe! It checks if the branch you're attempting to push to has been updated and sends you an error if it has. This gives you an opportunity to, as mentioned before, `fetch` the work and update your local repository.
+### Dangers and best practices
+
+Let's review the dangers we've addressed so far. I know, I know, it's scary stuff - but we have to be mindful or our coworkers might end up hating our guts! If you look back through this lesson you'll see a common thread. `amend`, `rebase`, `reset`, `push --force` are all especially dangerous when you're collaborating with others. <span id='dangers'>These commands can destroy work your coworkers have created</span>. So keep that in mind. When attempting to rewrite history, always check the dangers of the particular command you're using and follow these best practices for the commands we've covered:
+
+<span id='best-practices'></span>
+
+1.  If working on a team project, make sure rewriting history is safe to do and others know you're doing it.
+2.  Ideally, stick to using these commands only on branches that you're working with by yourself.
+3.  Using the `-f` flag to force something should scare you, and you better have a really good reason for using it.
+4.  Don't push after every single commit, changing published history should be avoided when possible.
+5.  Regarding the specific commands we've covered:
+    1.  For `git amend` never amend commits that have been pushed to remote repositories.
+    2.  For `git rebase` never rebase a repository that others may work off of.
+    3.  For `git reset` never reset commits that have been pushed to remote repositories.
+    4.  For `git push --force` only use it when appropriate, use it with caution, and preferably default to using `git push --force-with-lease`.
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1. Read the chapter on [Branches covered by git-scm](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell) if you want an even deeper dive into branches.
+2. Read [think-like-a-git](http://think-like-a-git.net/)
+    *   Take your time with this resource as well, it's very well written and will be very helpful in solidifying your understanding of Git.
```

Comment:
I would keep the merge conflicts reading here.

Reply:
Made the changes @wise-king-sullyman.

Reply:
Do you not want to update the Deeper Look at Git lesson as part of this PR?

Reply:
Oh ok I'll gladly do it @wise-king-sullyman

Reply:
Made the change @wise-king-sullyman

Reply:
It looks like you removed the entire bottom half of the Deeper Look at Git lesson? We want to keep the branches are pointers section, the relevant assignments, knowledge checks, and additional resources for that lesson.

Also you should update the lesson overview points.

Reply:
Ok @wise-king-sullyman. I'll add those back in. Can you point out which parts I should remove?

Reply:
All that should be removed from the Deeper Look at Git lesson is the stuff you added to the new remotes lesson.

Reply:
Ok